.aspect/workflows/terraform/workflows.tf

```diff
@@ -82,20 +82,22 @@ module "aspect_workflows" {
   bk_runner_groups = {
     # The default runner group is use for the main build & test workflows.
     default = {
-      agent_idle_timeout = 300
-      max_runners        = 5
-      min_runners        = 0
-      resource_type      = "default"
-      warming            = true
+      agent_idle_timeout_min = 5
+      max_runners            = 5
+      min_runners            = 0
+      queue                  = "default"
+      resource_type          = "default"
+      warming                = true
     }
     # The warming runner group is used for the periodic warming job that creates
     # warming archives for use by other runner groups.
     warming = {
-      agent_idle_timeout = 60
-      max_runners        = 1
-      min_runners        = 0
-      policies = { warming_manage : module.aspect_workflows.warming_management_policies["default"].arn }
-      resource_type = "default"
+      agent_idle_timeout_min = 1
+      max_runners            = 1
+      min_runners            = 0
+      policies               = { warming_manage : module.aspect_workflows.warming_management_policies["default"].arn }
```

Comment:
Should `.arn` to `.warn`?

Reply:
I guess that was only a whitespace change though 🤷

Reply:
`.arn` is correct here. refers to the AWS ARN of the policy.